integration-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname $0)"
+
+export GOBIN=$PWD/.tmp/bin
+mkdir -p $GOBIN
+go install ./go/cmd/swapi-server
+go install github.com/bufbuild/knit-go/cmd/knitgateway@v0.1.0
+
+curl -o ./.tmp/knitgateway.yaml https://raw.githubusercontent.com/bufbuild/knit-go/main/knitgateway.example.yaml
```

Comment:
Maybe we should just copy this or use a tagged version?

Reply:
I changed to use tagged version. As far as actually copying it into this repo, I kinda liked the property that this will verify that the config in knit-go correctly works with the swapi server, as it purports to do in the comments. Forking it into this repo would mean it could diverge in a way that isn't verified by any CI process.